pupil-auth/services/pin-generation.service.js

```diff
@@ -0,0 +1,70 @@
+const moment = require('moment')
```

Comment:
i look at this and see an emerging requirement for another standalone service - one that both the admin app and other APIs can call.  Seems like the only way to keep dependencies on key parts of logic from being duplicated.
@jon-shipley thoughts?

Reply:
@GuyHarwood 

Yes, I have thought something similar.   It certainly doesn't fit well in the auth service.

Reply:
thinking about the wider final design, the auth service should just do a lookup for the credentials, nothing more.  Meaning that the test user creds should always be there for pickup (if instance has test accounts enabled).  This is a separate piece of work